datadump/templatetags/datadump_tags.py

```diff
@@ -0,0 +1,32 @@
+from django import template
+from datadump.models import Post
+
+register = template.Library()
+
+
+def get_latest_posts(arg):
+    value = Post.objects.all()[:arg]
+    return value
+
+
+def get_most_tags(arg):
+    value = Post.tags.most_common()[:arg]
+    return value
+
+
+def get_most_comments(arg):
+    post_list = Post.objects.all().order_by('-comments')
+    post_counter = {}
+    for post in post_list:
+        if post in post_counter:
+            post_counter[post] += 1
+        else:
+            post_counter[post] = 1
+    popular_posts = sorted(post_counter, key=post_counter.get, reverse=True)
+    top = popular_posts[:arg]
+    return top
+
+
+register.filter('get_latest_posts', get_latest_posts)
+register.filter('get_most_tags', get_most_tags)
+register.filter('get_most_comments', get_most_comments)
```

Comment:
Newline gjithmone ne fund te file-it

Reply:
Nuk eshte shfaq mire per arsye se nuk eshte bo reload base.css, mundesh me hy n'incognito mode edhe tbon pastaj.

Reply:
Poo, ne incognito po bon super, kurse normal jo .. spo bon? Any idea why not?

Reply:
Mdoket eshte kur ja bon exec naj branch-i tjeter e bon save **base.css** nbaze te qatij branch-i, tani kur ta bojsh exec branch-un template-tags nuk ja bon reload **base.css** po e bon exec tu bazu ne ate file-un e vjeter. Mbas ni kohe rregullohet veq jo meniher, per kete arsye bon n'incognito.